### ISSUE TITLE
Missing Data of Export/Import of Taiwan

### ISSUE THREAD

**Issue by Ianhuangtw** [alexandersimoes/oec#57]
Hi, the data export from/import to Taiwan is not available.
Also, Taiwan would be replace as China when choosing other data presentation format such bilateral trade.
I guess it is also because of missing data.

Thanks for your help and I think this is a really beneficial and inspiring project.

**Comment by allenwlee** [comment]
i was curious about this as well. it's because their data ultimately comes from the United Nations[source explanation here](http://atlas.media.mit.edu/en/resources/data/), of which Taiwan is not a member.

**Comment by Ianhuangtw** [comment]
It must be, though Taiwan has been granted as an Observer in WHO. Wonder how this would adversely impact the research work in Taiwan.

**Comment by makmanalp** [comment]
@allenwlee @Ianhuangtw This explains it - straight from the comtrade people http://unstats.un.org/unsd/tradekb/Knowledgebase/Taiwan-Province-of-China-Trade-data?Keywords=China

**Comment by allenwlee** [comment]
Yes. Because China is a permanent member of the UN Security Council.

On Thursday, April 21, 2016, makmanalp <notifications@github.com> wrote:

> @allenwlee <https://github.com/allenwlee> @Ianhuangtw
> <https://github.com/Ianhuangtw> This explains it - straight from the
> comtrade people
> http://unstats.un.org/unsd/tradekb/Knowledgebase/Taiwan-Province-of-China-Trade-data?Keywords=China
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/alexandersimoes/oec/issues/57#issuecomment-213104738>
>


-- 
-allen.

Allen Lee 917-282-9938

**Comment by jspeis** [comment]
just ran into this and @davelandry explained its because Taiwan is not in the UN. might be nice to have a message explaining that the data for Taiwan is not available in the source

**Comment by q4w56** [comment]
In the comtrade data, the country tag "Other Asia" practically represent Taiwan.[[1]](http://unstats.un.org/unsd/tradekb/Knowledgebase/Taiwan-Province-of-China-Trade-data) So I think it is good to make the tag "Other asia" accessible from web api.

After my glimpse on the code base, I think it can be be achieved by giving "other asia" a 3 letter id, in the table attr_country [(table link)](http://18.85.28.32/static/db/csv/).

id | id_2char | id_3char | id_num | color | comtrade_name | id_old
-- | -- | -- | -- | -- | -- | --
asxxb | NULL | NULL | 490 | #c8140a | Other Asia, nes | 251

That is, change the value of "id_3char" from "NULL" to "xxb".
And then other asia should be accessible under the tag "xxb".
